internal/syscall.go

```diff
@@ -91,6 +91,19 @@ func BPFProgDetach(attr *BPFProgDetachAttr) error {
 	return err
 }
 
+const (
+	BPF_STATS_RUN_TIME = iota
+)
+
+type BPFEnableStatsAttr struct {
+	StatsTyp uint32
+}
+
+func BPFEnableStats(attr *BPFEnableStatsAttr) error {
+	_, err := BPF(BPF_ENABLE_STATS, unsafe.Pointer(attr), unsafe.Sizeof(*attr))
```

Comment:
I would like to see a feature test for this function. [See objectNames](https://github.com/cilium/ebpf/blob/master/syscalls.go#L399) as an example.